exercises/css-fonts/styles.css

```diff
@@ -0,0 +1,18 @@
+@font-face {
+  font-family: 'Architects Daughter', cursive;
+  src: url(https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Cairo:wght@300&display=swap);
+}
+
+@font-face {
+  font-family: 'Cairo', sans-serif;;
+  src: url(https://fonts.googleapis.com/css2?family=Cairo:wght@300&display=swap);
+}
```

Comment:
This can be removed, this work is already being done by the `link` element for the font families